README.md

```diff
@@ -0,0 +1,151 @@
+<p align="center">
```

Comment:
- [ ] Kindly, go to the frontend repo to see the [review ](https://github.com/Kaiserabbas/motorcycle_booking_front_end/pull/21#pullrequestreview-1814587538)comment. 👍